letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -912,6 +912,34 @@ OldVenvExists() {
     [ -n "$OLD_VENV_PATH" -a -f "$OLD_VENV_PATH/bin/letsencrypt" ]
 }
 
+# Given python path, version 1 and version 2, check if version 1 is outdated compared to version 2.
+# Unofficial versions (eg. 0.28.0.dev0) will be treated specifically by printing "UNOFFICIAL".
```

Comment:
Well, an unofficial version of $2 will say UNOFFICIAL. An unofficial version of $3 will say UP_TO_DATE. I think those are reasonable choices, but we should bring this comment up to date.

Reply:
I update the inline comment.